backend/src/v5/middleware/dataConverter/inputs/users.js

```diff
@@ -134,7 +139,12 @@ Users.validateForgotPasswordData = async (req, res, next) => {
 
 		try {
 			const usernameOrEmail = req.body.user;
-			const { user } = await getUserByUsernameOrEmail(usernameOrEmail, { user: 1, _id: 0 });
+			const { user, customData: { sso } } = await getUserByUsernameOrEmail(usernameOrEmail, { user: 1, _id: 0, 'customData.sso': 1 });
+
+			if (sso) {
```

Comment:
I don't think we should be throwing error?

It should return as usual, just don't send email -this is(or at least, should be) the behaviour when someone is trying to trigger forgot password with an invalid email address/username

Reply:
Its being caught in line 151 and returns OK